RobotApplication.java

```diff
@@ -1 +1,28 @@
+package com.company;
 
+public class RobotApplication {
+
+    public static void main(String[] args) {
+        RobotConnectionManager robotConnection = null;
+        moveRobot(robotConnection, 4, 5);
+    }
+
+    public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY) {
```

Comment:
is this method work on stepic correctly?

Reply:
where  are you close your connection?

Reply:
Sorry.
I have not downloaded the latest version of the code. I did it